build.sbt

```diff
@@ -176,26 +102,27 @@ lazy val plugin = project
         )
         .value
     },
-    scalacOptions ++= scalacCompilerParams,
     mimaFailOnNoPrevious := false,
   )
 
-playBuildRepoName in ThisBuild := "twirl"
-playBuildExtraTests := {
-  (scripted in plugin).toTask("").value
-}
-playBuildExtraPublish := {
-  (PgpKeys.publishSigned in plugin).value
-}
+// FIXME: how to port this?!
+// playBuildRepoName in ThisBuild := "twirl"
+// playBuildExtraTests := {
+//   (scripted in plugin).toTask("").value
+// }
+// playBuildExtraPublish := {
+//   (PgpKeys.publishSigned in plugin).value
+// }
```

Comment:
This is not necessary since `pgp` signing will now happen on bintray. Not sure about the `scripted` part... looking.

Reply:
removed, still checking the scripted part